### DISCUSSION TITLE
Multimodal journaling

### DISCUSSION THREAD

**Discussion by sallirom** [zim-desktop-wiki/zim-desktop-wiki#1896] [Ideas]
Hi, I am a heavy journal user. I find it extremely useful. Recently I am adding to my weekly routines a week resume. For that, it would be very handy to have a  way to add a weekly journal page easily. Now I have to add it manually or to go to preferences, change it to weekly, write my stuff and then change again to daily. 

Maybe there is a way to do this easily and I just haven´t figured out how, excuse me if that would be the case.

**Comment by jaap-karssenberg** [comment]
There is no obvious way to do this.

What you could do is modify the template for journal pages such that each
daily page includes a link to the corresponding weekly page. That way you
can access the weekly page by clicking the link and go from there



Op ma 31 jan. 2022 11:12 schreef sallirom ***@***.***>:
